pyramid_zipkin/zipkin.py

```diff
@@ -199,3 +199,38 @@ def __exit__(self, _exc_type, _exc_value, _exc_traceback):
             binary_annotations=self.binary_annotations,
             span_id=self.span_id,
         )
+
+
+def zipkin_span(
+    service_name,
+    span_name,
+    binary_annotations=None,
+):
+    """Decorator utility for logging a function as a Zipkin span.
+    Note that the decorator doesn't accept timestamp annotations.
+    Any timestamps included in the decorator would likely be way off,
```

Comment:
Hmm. Yeah this is tough to describe. Maybe something like:

> Note that the decorator doesn't accept timestamp annotations.
> "Annotations" in Zipkin are timestamped events, so statically defining them in code doesn't make sense.

I dunno :confounded:. For these types of things, I would lean toward less-is-more for comments and expect that the reader will understand more if they think about it for a moment themselves than risk confusing them.